client/customloaders/customfileloader/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "customfile-loader",
+  "version": "0.0.2",
+  "description": "custom file loader module for us",
+  "files": [
+    "index.js"
+  ],
+  "dependencies": {
+    "bluebird": "^2.10.2",
+    "fs-extra": "^0.30.0",
+    "loader-utils": "~0.2.5",
+    "mime": "~1.3.4"
```

Comment:
This should pin down the exact package versions as the `client/package.json` file does.